client/src/components/InfoAndLinks.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import RecyclingInfo from '../components/RecyclingInfo';
 import PropManagerLinks from '../components/PropManagerLinks';
+import Footer from '../components/Footer';
 
 const InfoAndLinks = () => (
   <div>
     <RecyclingInfo />
     <PropManagerLinks />
+    <Footer />
```

Comment:
I think a better place for the `<Footer />` would be in Home.js after the main content `<Grid/>`  closes.